### ISSUE TITLE
Change the way the Disk sensors are created in HA [Feature]

### ISSUE THREAD

**Issue by thanomichalis** [tomaae/homeassistant-openmediavault#142]

**Comment by tomaae** [comment]
serial number is a no-go. some controllers or usb dont provide serial. same for mc which dont have serial at all.
it has to be something that is unique and is always provided.

**Comment by thanomichalis** [comment]
@tomaae i see. However it is an issue, cause it is not only the history and statistics, but also the fact that in HA any automation regarding the disks, cannot be reliable since the sensors, are becoming unavailable or they have the wrong data.
We may use the UUID though.

**Comment by tomaae** [comment]
yes, I was thinking of UUID too, but it needs confirming and testing that UUID stays the same while dev changes.

**Comment by thanomichalis** [comment]
@tomaae I am not a programmer nor familiar with what exactly should i do, but if you give me some script to add it in my HA installation on rpi4, i might be able to run some tests.

**Comment by Henkow** [comment]
It for sure stays the same. I also would like UUID instead. The current implementation doesn't make it possible to rename the drives to what they actually are. I for instance can't name the OS drive as SSD since it may go from sda to sdd upon reboot.

Speaking of this, the same issue is the case with containers running. Would be nice if "OMV Compose" could be removed from the name so it only display what service is running, would make it more clean as "OMV Compose" takes up so much space. Can't really hardcode any since the container running a given service may go from omv_compose_5 to omv_compose_3 upon reboot.

**Comment by Hantick** [comment]
Having the same issue, my HDDs are swapping names between each other and SSD
